docs/src/main/tut/docs/built-in-supported-types.md

```diff
@@ -36,6 +36,7 @@ Additionally, PureConfig also handles the following collections and composite Sc
   [Configurable Converters](configurable-converters.html));
 - `shapeless.HList`s of elements whose type is on this list;
 - case classes;
+- classes with only public `val` and `var` parameters in their constructor (Scala 2.11+);
```

Comment:
We get this from free with the new Shapeless version, but perhaps we should also mention it in the change log.

Reply:
Yes, I agree. Lately we've only been updating the CHANGELOG just before each version release, though.